WeakAurasOptions/RegionOptions/DynamicGroup.lua

```diff
@@ -75,6 +75,28 @@ local function createOptions(id, data)
   local options = {
     __title = L["Dynamic Group Settings"],
     __order = 1,
+    groupIcon = {
```

Comment:
Add a tooltip, please.

Reply:
used `desc = L["Set Thumbnail Icon"],` for dynamicgroup & group groupIcon if that's ok